src-built-in/adapters/indexedDBAdapter.js

```diff
@@ -178,6 +190,11 @@ const IndexedDBAdapter = function () {
 		Logger.system.debug("IndexedDBAdapter.delete for key=" + combinedKey);
 		console.debug("IndexedDBAdapter.delete for key=" + combinedKey);
 
+		if (!this.db) {
```

Comment:
In general, I think I would put the queue pushes before the logging statements, so the delete of a key doesn't get logged twice, even though it only gets executed once.